src/actions/offices.js

```diff
@@ -13,54 +13,59 @@ export const fetchOffices = offices => {
 
 const fetch = require("node-fetch");
 
-
 export const setCandidates = candidates => {
   return {
     type: CANDIDATES,
     candidates
-  }
-}
+  };
+};
 
 export const fetchCandidatesInAnOffice = (office_id, token) => {
-  return fetch(`${BASE_URL}/offices/${office_id}/candidates`, {
-    method: "GET",
-    headers: {
-      "Content-Type": "application/json",
-      authorization: `Bearer ${token}`
-    }
-  })
-    .then(res => res.json())
-}
+  return axios
+    .get(`${BASE_URL}/offices/${office_id}/candidates`, {
+      headers: {
+        "Content-Type": "application/json",
+        authorization: `Bearer ${token}`
+      }
```

Comment:
Put this one in a helper function so that you can reuse across all the actions. The functions will take parameters such as `method`, `payload/data`, `endpoint`.